projects/ngx-charts-on-fhir/src/lib/data-layer/data-layer-color.service.ts

```diff
@@ -27,9 +27,8 @@ export class DataLayerColorService {
     line.backgroundColor = color + '33'; // temporary dirty hack to set opacity. assumes color is in 6-character hex format.
   }
 
-  getAnnotationColor(annotation: any): string | undefined {
-    const color = annotation.backgroundColor;
-    if (typeof color === 'string' && !color.includes('33')) {
+  addTransparency(color: string | undefined): string | undefined {
+    if (typeof color === 'string' && !color.endsWith('33')) {
```

Comment:
This will not work correctly for colors that end with `33` but have no alpha channel (like `#FFFF33`) or colors that have an alpha channel other than `33` (like `#FFFFFF66`) or colors that use 3-digit hex or rgb or any other type of color string. We really should be using a library for color manipulation, so I'll just approve this PR and create a new issue.